sdk/src/main/java/io/customer/sdk/data/store/BuildStore.kt

```diff
@@ -17,7 +17,7 @@ interface BuildStore {
     // Android SDK Version: 21
     val deviceOSVersion: Int
 
-    // Device locale: en
+    // Device locale: en-US
```

Comment:
From conversations with LJ, I believe the backend is looking for `en_US` (notice underscore).